Gruntfile.js

```diff
@@ -19,7 +19,7 @@ module.exports = function(grunt) {
     },
     watch: {
       scripts: {
-        files: ['views/*.lavender'],
+        files: ['views/*.lavender','views/layouts/*.lavender'],
```

Comment:
You can tighten this up with a single call to watch 'views/**/*.lavender'

This watches lavender files in the views directory and any of its subdirectories.